cirq/ops/controlled_operation_test.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+import numpy as np
+import cirq
+from cirq import protocols
+
+
+def test_controlled_operation_init():
+    cb = cirq.NamedQubit('ctr')
+    q = cirq.NamedQubit('q')
+    g = cirq.Gate()
+    v = cirq.GateOperation(g, (q,))
+    c = cirq.ControlledOperation(cb, v)
+    assert c.sub_operation == v
+    assert c.control == cb
+    assert c.qubits == (cb, q)
+
+    eq = cirq.testing.EqualsTester()
+    eq.add_equality_group(c, c.with_qubits(cb, q))
```

Comment:
Just assert that `c == c.with_qubits(cb, q)` since you already have a test doing equality group stuff.

Reply:
ack